Upgrade geoalchemy

pyramid_oereb seems to be currently not compatible with geoalchemy 0.9.0, in the CI tests there are errors like the following:

`E       psycopg2.errors.SyntaxError: syntax error at or near "limit"
E       LINE 1: ...e_limit ON pyramid_oereb_main.real_estate USING GIST (limit)
E                                                                        ^

/usr/local/lib/python3.7/dist-packages/sqlalchemy/engine/default.py:717: SyntaxError

The above exception was the direct cause of the following exception:

    @pytest.mark.run(order=-2)
    def test_create_standard_db():
        assert pyramid_oereb_test_yml is not None
        from pyramid_oereb.standard import create_tables_from_standard_configuration
>       create_tables_from_standard_configuration(pyramid_oereb_test_yml)

tests/test_standard_db_creation.py:11: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
pyramid_oereb/standard/__init__.py:107: in create_tables_from_standard_configuration
    main_base_class.metadata.create_all(main_schema_engine)
`

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,6 @@
 PyPDF2==1.26.0
 filetype==1.0.7
-geoalchemy2==0.8.5 # rq.filter: <0.9
+geoalchemy2==0.10.2
 pyramid==1.10.8 # rq.filter: <2
 pyramid-debugtoolbar==4.9
 shapely==1.6.4.post1  # rq.filter: ==1.6.4.post1

